CMSgov/mint-app
# [EASI-2483] - Mac user homepage

# EASI-2483

## Changes and Description

- Added job code and util for MINT Mac Users
- Updated home page query for additional fields if MAC user
- Configured specific columns for MAC users
- Added ability to query documents as a MAC user
- Button removed to start a new model plan

![Screenshot 2022-12-06 at 1 25 23 PM](https://user-images.githubusercontent.com/95709965/205991760-b000b9d4-f21f-4918-9176-3be957002b1f.png)



## How to test this change

- Log in with MAC role with dev login
- Verify table matches figma
- Verify access controls are in place so that MAC user can't edit anything

## Notes

Currently there is no GQL query filter for getting all model plans that have CRTDLs (which is a part of the AC)
This will be addressed later in a full stack ticker [EASI-2503](https://jiraent.cms.gov/browse/EASI-2503)

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
